app/src/main/res/values/strings.xml

```diff
@@ -84,4 +84,8 @@
   <string name="message_share">Check out #Monotweety! %s #Android</string>
 
   <string name="transition_name_logo" translatable="false">transition.logo</string>
+
+  <!-- Notification -->
+  <string name="notification_channel_editor_name">Editor</string>
```

Comment:
**[Android Lint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
Warning: The resource R.string.notification_channel_editor_name appears to be unused [UnusedResources]